dist/api_data.py

```diff
@@ -1910,6 +1910,12 @@ def __ge__(self, other):
         Config('checkpoint_cleanup', 'false', r'''
             if true, checkpoint cleanup thread is triggered to perform the checkpoint cleanup''',
             type='boolean'),
+        Config('checkpoint_crash_point', '-1', r'''
+            non-negative number between 0 and 1000 will trigger a controlled crash during the
+            checkpoint process. Lower values will trigger crashes in the initial phase of
+            checkpoint, while higher values will result in crashes in the final phase of the
+            checkpoint process ''',
```

Comment:
Nitpick: trailing space between `process` and `'''`.

Reply:
Fixed in d58ab509ef6a2652ed8bdd2d7578e0f492f06916